os/net/mac/mac-sequence.h

```diff
@@ -45,22 +45,31 @@
 #ifndef MAC_SEQUENCE_H
 #define MAC_SEQUENCE_H
 
+enum mac_sequence_result {
+  MAC_SEQUENCE_NO_DUPLICATE = 0,
+  MAC_SEQUENCE_DUPLICATE,
+  MAC_SEQUENCE_ERROR
```

Comment:
I think C99 allows a trailing comma after the last enum-member (allows extending enums without affecting git blame).